songs/Angbatssang.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Ångbåtssång
+author: Otto Lindblad
+midiFiles: []
+svgFiles:
+  - songs/Angbatssang-1.svg
+  - songs/Angbatssang-2.svg
+  - songs/Angbatssang-3.svg
+  - songs/AngbatssangEnd.svg
+---
```

Comment:
Suggestion: ignore md files and see them as build artifacts. Instead, have just a yaml file with title and author, from which a script adds svgFiles and midiFiels